bcda/auth/client/ssas_test.go

```diff
@@ -166,7 +186,6 @@ func (s *SSASClientTestSuite) TestCreateGroupTable() {
 	}{
 		{fnInput: []string{"1", "name", "", `{ "ID": 123456 }`}, header: http.StatusCreated, env: EnvVars{}, errorExpected: false, errorMessage: ""},
 		{fnInput: []string{"1", "name", "", `{ "ID": 123456 }`}, header: http.StatusCreated, env: EnvVars{SSAS_URL: "localhost"}, errorExpected: true, errorMessage: "Post \"localhost/group\": unsupported protocol scheme \"\""},
-		{fnInput: []string{"\"", "name", "", `{ "ID": 123456 }`}, header: http.StatusCreated, env: EnvVars{SSAS_URL: "\n"}, errorExpected: true, errorMessage: "invalid input for new group_id \": parse \"\\n/group\": net/url: invalid control character in URL"},
```

Comment:
I removed a few test cases that tested for a specific http.Client error, the failure conditions for which now generate a SSAS client error instead. This error is tested in separate unit tests.